doc/modules/mod_event_pusher.md

```diff
@@ -0,0 +1,36 @@
+### Module Description
+This module is a generic interface for event pushing backends.
+It defines a single hook, `push_event/3` that forwards the event to all registered backends.
+Each backend decides how and if to handle the event in its `push_event/2` implementation.
+
+The events are standardized as records that can be found in `mod_event_pusher_events.hrl` file.
+Common events like user presence change (offline and online), chat and groupchat messages (incoming and outgoing) are already hooked up to the frontend via `mod_event_pusher_hook_translator` which is a proxy between various hooks and `push_event/3` hook handler.
```

Comment:
Common events like user presence changes (offline and online), chat and groupchat messages (incoming and outgoing) are already hooked up to the frontend via `mod_event_pusher_hook_translator` which is a proxy between various hooks and `push_event/3` hook handler.